solutions/devsprint-ana-nogal-1/DeliveryApp/Screens/LoadingView/LoadingView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  LoadingView.swift
+//  DeliveryApp
+//
+//  Created by Luiza on 26/04/22.
+//
+
+import UIKit
+
+final class LoadingView: UIView {
+
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Buscando endereços..."
+        label.font = UIFont.systemFont(ofSize: 16, weight: .bold)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    private lazy var activityIndicator: UIActivityIndicatorView = {
+        let indicator = UIActivityIndicatorView(style: .large)
+        indicator.startAnimating()
+        indicator.translatesAutoresizingMaskIntoConstraints = false
+        return indicator
+    }()
+    
+    private lazy var stackView: UIStackView = {
+        let stack = UIStackView()
+        stack.axis = .vertical
+        stack.alignment = .center
+        stack.spacing = 14
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        return stack
+    }()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configViews()
+        buildViews()
+        setupConstraints()
+    }
+        
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+
+extension LoadingView: ViewConfiguration {
+    internal func configViews() {
```

Comment:
O Access Control por padrão no Swift é o internal... no need to add it. 😄 
```suggestion
    func configViews() {
```

Reply:
done